slack/slice_test.go

```diff
@@ -0,0 +1,87 @@
+package slack
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestFindReturnsTrueWhenSliceContainsGivenString(t *testing.T) {
+	cases := []struct {
+		n string
+		h []string
+	}{
+		{"Caninos", []string{"Simularbre", "Caninos", "Noctali", "Gobou"}},
+		{"Caninos", []string{"Caninos"}},
+	}
+
+	for _, testCase := range cases {
+		if find(testCase.n, testCase.h) != true {
+			t.Errorf("Expected true, got false when finding %s in %s", testCase.n, testCase.h)
+		}
+	}
+}
+
+func TestFindReturnsFalseWhenSliceDoesNotContainGivenString(t *testing.T) {
```

Comment:
I would merge this test with the previous one, and add a `label` and an `expectation` field in the case structure.